syncer/serf/agent.go

```diff
@@ -54,19 +52,12 @@ func Create(conf *Config, logOutput io.Writer) (*Agent, error) {
 }
 
 // Start agent
-func (a *Agent) Start(ctx context.Context) {
```

Comment:
why delete context management

Reply:
@tianxiaoliang The use of context here is intended to handle the graceful exit of the serf. But it has been processed in syncer/service.go (s *Server) Stop(), I don't think it is necessary to start goroutines here and repeat it. see https://github.com/apache/servicecomb-service-center/blob/master/syncer/server/server.go#L99